src/main/java/com/exasol/sql/ValueTableRow.java

```diff
@@ -19,7 +22,7 @@
      */
     public ValueTableRow(final Fragment root, final ValueExpression... expressions) {
         super(root);
-        this.expressions = Arrays.asList(expressions);
+        this.expressions = asList(expressions);
```

Comment:
Static method access through class should be explicit in this case. Please correct. Looks like IntelliJ is changing that too.

Reply:
fixed